src/FluentMigrator.Runner.SQLite/Generators/SQLite/SQLiteQuoter.cs

```diff
@@ -22,5 +22,18 @@ public override string QuoteSchemaName(string schemaName)
         {
             return string.Empty;
         }
+
+        protected override string FormatByteArray(byte[] value)
+        {
+            var hex = new System.Text.StringBuilder((value.Length * 2) + 2);
```

Comment:
Shouldn't this be a `+ 3`? The `X` and both single quotes `'`.

Reply:
Yes! My bad! Amending now.